tubemq-client-twins/tubemq-client-cpp/src/tubemq_config.cc

```diff
@@ -32,6 +32,112 @@ using std::set;
 using std::stringstream;
 using std::vector;
 
+
+
+TubeMQServiceConfig::TubeMQServiceConfig() {

```

Comment:
It is better to use the initialization list for initialization.

Reply:
I think it’s better to keep the current initialization mode: if the initialization is done in the .h file, the internal const_config.h file will be exposed to the outside, which will make the API definitions of the user contact not concise enough